### DISCUSSION TITLE
Signing application so I can cask installed apps

### DISCUSSION THREAD

**Discussion by kaihendry** [Homebrew/discussions#1553] [Everyday usage]
I've look https://brew.sh/, but I am still confused how I am supposed to sign applications.

https://github.com/flameshot-org/flameshot/issues/1641#issuecomment-849237508

1. Why can't home brew sign it?
2. If I have to sign it, how do I set it up?
3. Where is this all documented? Is there an IRC channel?

**Comment by carlocab** [comment]
https://docs.brew.sh/FAQ#the-app-cant-be-opened-because-it-is-from-an-unidentified-developer
